### DISCUSSION TITLE
I could not set the MSBuildToolVersion value as NET461

### DISCUSSION THREAD

**Discussion by parthipanr** [cake-build/cake#3091] [Q&A] [answered]
### What You Are Seeing?
I could not build the .NET Standard 2.0 version project using cake script and Need to Automate the .NET Standard 2.0. There is no options to set MSBuildToolVersion value as NET461. (NET Standard 2.0 target version is NET461). 

### What version of Cake are you using?
Latest version
### Are you running on a 32 or 64 bit system?
64 bit 
### What environment are you running on?  Windows? Linux? Mac?
windows
### How Did You Get This To Happen? (Steps to Reproduce)
Create the project in VS 2017 15.3 version.
Use the below script for build. 
The target version not set exception will thrown.  

### Output Log

GIST LINK - Please create a gist and link to that gist here

OR

~~~sh
PLACE LOG CONTENT HERE
~~~

**Comment by patriksvensson** [comment]
`MSBuildToolVersion` is the version of MSBuild that you want to use and not how you set the target framework.

**Comment by parthipanr** [comment]
Yes, I want to set the MSBuild version as NET461 using MSBuildToolVersion enum. Can you please suggest any other way to achieve it, if any?

**Comment by patriksvensson** [comment, marked as answer]
@parthipanr So you want to use `MSBuild 15` in other words. Pass [VS2017](https://cakebuild.net/api/Cake.Common.Tools.MSBuild/MSBuildToolVersion/FC0BFFE4) which will correspond to the `MSBuildVersion` [MSBuild15](https://cakebuild.net/api/Cake.Common.Tools.MSBuild/MSBuildVersion/9B1D06EA).
